res/database/Default/database_people_fictional.xml

```diff
@@ -6,7 +6,8 @@
 			<first_name>Babsi</first_name>
 			<last_name>Angora</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="2" birthday="-20" deathday="" country="D" fictional="1" />
+			<!-- relative birthdate not advisable here, she is used in some programmes with fixed dates (and described as an 'ex-model'); the programmes with flexible dates should fit well too now -->
```

Comment:
I agree in general that more (most) fictional programs should have relative dates, but this character was already used for some fixed-date programs referring to real-life events. First one is 1980, so "ex-model" with 29 kinda is a joke in itself already, and leaves enough wiggle-room for other start dates. Generally for the story-telling aspect of the game, attention definitely has to be paid to that, and in most cases, staff of fixed-date programs has to be almost completely separated from staff of dynamic-date programs. I'll go through the fictional programs eventually and see if more fine-tuning has to be done in that regard.

Reply:
So possibly add a TODO here? It may not be a problem to make the Babsi-programmes reletive, as well. To me, it seems she is quite an important cast character, which makes a relative age more appropriate

Reply:
marking the comment with TODO, but I don't see a problem here. She had long already been used in a fixed-date program, and the dynamic ones where she's in also already have boundaries.